src/test/resources/application.properties

```diff
@@ -4,3 +4,4 @@ REGISTERED_EMAIL_ADDRESS_UPDATE_FILING_DESCRIPTION=Registered Email Address upda
 
 allowed.company.statuses = active, liquidation, receivership, voluntary-agreement, insolvency-proceedings, administration
 allowed.company.types = private-unlimited, ltd, plc, private-limited-guarant-nsc-limited-exemption, private-limited-guarant-nsc, private-unlimited-nsc, private-limited-shares-section-30-exemption, llp
+env.name=test
```

Comment:
is this needed?

Reply:
Good spot - no, it was needed for a previous version of this during development but not any more.